internal/provider/provider.go

```diff
@@ -156,6 +157,14 @@ func azureProvider() *schema.Provider {
 				Description: "This will disable the Terraform Partner ID which is used if a custom `partner_id` isn't specified.",
 			},
 
+			"default_name": resourceName.SchemaResourceName(),
+
+			"default_naming_prefix": resourceName.SchemaResourceNamePrefix(),
+
+			"default_naming_suffix": resourceName.SchemaResourceNameSuffix(),
+
+			"naming_caf_enabled": resourceName.SchemaResourceNameCafEnabled(),
```

Comment:
Can we remove this field and its schema, since it's not used.

Reply:
removed.